apps/minifront/src/components/dashboard/transaction-table.tsx

```diff
@@ -6,67 +6,65 @@ import {
   TableHeader,
   TableRow,
 } from '@penumbra-zone/ui/components/ui/table';
-import { Link, LoaderFunction, useLoaderData } from 'react-router-dom';
-import { getAllTransactions, TransactionSummary } from '../../fetchers/transactions';
+import { Link, LoaderFunction } from 'react-router-dom';
 import { throwIfPraxNotConnectedTimeout } from '@penumbra-zone/client';
 import { shorten } from '@penumbra-zone/types/src/string';
+import { useStore } from '../../state';
+import { memo, useEffect } from 'react';
+import { TransactionSummary } from '../../state/transactions';
 
-export const TxsLoader: LoaderFunction = async (): Promise<TransactionSummary[]> => {
+export const TxsLoader: LoaderFunction = async (): Promise<null> => {
   await throwIfPraxNotConnectedTimeout();
-  return await getAllTransactions();
+  return null;
 };
```

Comment:
notConnected is already handled by the top-level layoutloader. if that's all that remains here, the loader can be deleted.

Reply:
Good catch! Fixed